server/src/main/java/io/github/axisangles/ktmath/Vector3.kt

```diff
@@ -0,0 +1,92 @@
+@file:Suppress("unused")
+
+package io.github.axisangles.ktmath
+
+import kotlin.math.atan2
+import kotlin.math.sqrt
+
+data class Vector3(val x: Float, val y: Float, val z: Float) {
```

Comment:
No "distance" method? It's pretty standard for most implementations to have one.

Reply:
There might be some methods missing. I felt (b - a).len() was sufficient but maybe not. Same goes for some methods in the matrix library like angleTo, and angle. if you think of any features you want to see, let me know and I'll add them shortly.

Reply:
`(b - a).len()` is definitely sufficient, I just figure shortcuts like `.distance(Vector3 other)` are good to have as it'll be used really frequently

Reply:
I personally coming from a math side of things prefer ``(b - a).len()`` for representing distance. They are as short and they are more explicit.

Reply:
I think it's probably worth it though I suppose someone or I can always add these additions in the future.